### PR TITLE
Sirtutim section

### PR DESCRIPTION
**Pull request by davgur**

### CODE REVIEW

File: src/components/Layout/MenuItems.js [hunk 1]
```diff
@@ -18,16 +18,13 @@ const ITEMS = [
   'sources',
   'likutim',
   'events',
-  // 'books',
   'topics',
   'publications',
   'music',
   'simple-mode',
+  'sketches',
   // 'project-status',
```
Comment (bbfsdev) [inline src/components/Layout/MenuItems.js:26]: Remove project-status?

File: src/helpers/Api.js [hunk 2]
```diff
@@ -2,7 +2,7 @@ import axios from 'axios';
 import { MY_NAMESPACE_LABELS, MY_NAMESPACE_PLAYLIST_EDIT, MY_NAMESPACE_PLAYLISTS } from './consts';
 
 const API_BACKEND             = process.env.REACT_APP_API_BACKEND;
-const ASSETS_BACKEND          = process.env.REACT_APP_ASSETS_BACKEND;
+const ASSETS_BACKEND          = 'https://kabbalahmedia.info/assets/';//process.env.REACT_APP_ASSETS_BACKEND;
```
Comment (bbfsdev) [inline src/helpers/Api.js:5]: You should use here process.env, right?

File: src/helpers/Api.js [hunk 3]
```diff
@@ -21,6 +21,8 @@ export const chroniclesUrl            = path => `${CHRONICLES_BACKEND}${path}`;
 export const chroniclesBackendEnabled = CHRONICLES_BACKEND !== undefined;
 
 export class Requests {
+  static encode = encodeURIComponent;
```
Comment (bbfsdev) [inline src/helpers/Api.js:24]: Where encode is used? Remove?

**Review by bbfsdev — CHANGES_REQUESTED** [review]
There is some error in this url: https://staging-archive.kabbalahmedia.info/he/lessons/series/cu/SGV2aoYI?activeTab=sketches
image

1. On the left side שגיאת שרת
2. Sketches tabs are broken (text instead of thumbnails)


Is this filter new?
image

Should we also support this for search?

File: src/components/Sections/Sketches/ZipFileModal.js [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+      className="image-gallery-fullscreen-button"
+      icon={isFullscreen ? 'compress' : 'expand'}
+      onClick={onClick}
+    />
+  );
+
+  return (
+    <>
+      <FallbackImage
+        src={assetUrl(path.substring(8))}
```
Comment (bbfsdev) [inline src/components/Sections/Sketches/ZipFileModal.js:64]: Feels fragile, do we do that to extract the UID?
Do we do that in all places?

Comment (davgur) [reply src/components/Sections/Sketches/ZipFileModal.js:64]: I was take it from prev code. From unzip service return only path and size of file. Its not take UID, its remove "/asserts" from image path

File: src/helpers/Api.js (hunk 2 above)
Comment (davgur) [reply src/helpers/Api.js:5]: yes, will change it for prod, I cant deploy asserts backend for staging

File: src/components/Layout/MenuItems.js (hunk 1 above)
Comment (davgur) [reply src/components/Layout/MenuItems.js:26]: done

File: src/components/Sections/Sketches/MainPage.js [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+  const pageNo   = useMemo(() => getPageFromLocation(location) || 1, [location]);
+
+  let wipAll;
+  const zipIdsForFetch = cus?.map(x => x.files).flat()
+    .filter(x => MediaHelper.IsImage(x) && isZipFile(x))
+    .map(x => x.id)
+    .filter(id => {
+      const x             = getZipById(id) || false;
+      const { wip, data } = x;
+      if (wip) wipAll = !!wip;
```
Comment (bbfsdev) [inline src/components/Sections/Sketches/MainPage.js:56]: if (wip) wipAll = true

Much clearer.

**Review by bbfsdev — COMMENTED** [review]
What are those sketches:
image

https://staging-archive.kabbalahmedia.info/he/sketches?content_type=VIDEO_PROGRAM_CHAPTER&topics=0db5BBS3

File: src/components/Sections/Sketches/MainPage.js (hunk 5 above)
Comment (davgur) [reply src/components/Sections/Sketches/MainPage.js:56]: done

File: src/components/Sections/Sketches/ZipFileModal.js [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+import { imageGalleryItem } from '../../Pages/Unit/widgets/UnitMaterials/helper';
+import FallbackImage from '../../shared/FallbackImage';
+import { assetUrl } from '../../../helpers/Api';
+
+const ZipFileModal = ({ id, path }) => {
+  const [open, setOpen] = useState(false);
+
+  const getZipById = useSelector(state => assets.nestedGetZipById(state.assets));
+  const language   = useSelector(state => settings.getLanguage(state.settings));
+  const ref        = useRef();
```
[the remark this answers is not in the document]
Comment (davgur) [reply src/components/Sections/Sketches/ZipFileModal.js:18]: its used on line 77 for gallery library

File: src/helpers/Api.js (hunk 3 above)
Comment (davgur) [reply src/helpers/Api.js:24]: its not mine, used on the line 65/68

**Review by bbfsdev — APPROVED** [review]

**Pull request merged**